test/test_jit.py

```diff
@@ -1176,9 +1196,22 @@ def forward(self, x):
         %w_dequant = aten::dequantize(%w_quant)
         %r = aten::conv2d(%a_dequant, %w_dequant, %b, %stride, %padding, %dilation, %groups)
         return (%r)"""
-        graph = parse_ir(conv_input_str)
-        torch._C._jit_pass_insert_prepack_unpack(graph)
-        FileCheck().run(conv_input_str, graph)
+
+        conv_per_channel = """
```

Comment:
This test is useless, it just repeats the pattern from the code. If the pattern happens to be buggy, this test will still pass. Can we rewrite it in a similar way to the test above? If not, it's better to just remove it.

Reply:
I think even the above code is pretty similar to just repeating the pattern, maybe only way to test this properly is through  integration test.

Reply:
should we just remove this test then?

Reply:
By the code above I meant the code where you create actual modules - these tests are fine. But here we're literally creating the same IR as in the pattern - it's fine to delete this if there is no way to rewrite it like the code with modules.

Reply:
reason we do this is in Linear we are matching CallFunction and in Conv2d we are matching aten::conv2d. I think we can write this test in similar ways as the Linear test.